s3_management/manage.py

```diff
@@ -226,7 +227,7 @@ def to_legacy_html(
         out: List[str] = []
         subdir = self._resolve_subdir(subdir)
         is_root = subdir == self.prefix
-        for obj in self.gen_file_list(subdir):
+        for obj, _ in self.gen_file_list(subdir):
```

Comment:
I am not as familiar with the legacy html, but perhaps it too should have the checksums?